### ISSUE TITLE
Issue with submitting slurm job an omega. 

### ISSUE THREAD

**Issue by jmcclena** [pabloprf/MITIM-fusion#42]
Running portals on omega runs fine. However, when I tried to have portals submit a slurm job, the job got stuck in the queue.  I suspect there is some issue with the resources requested. 

@pabloprf to debug this, is there a way I can see the exact slurm command MITIM submitted to the queue?

**Comment by pabloprf** [comment]
Yes, you should be able to go to the scratch folder where the job is supposed to run, and look at file `mitim_bash.src` which is what's submitted as SBATCH.
Please take a look and let me know.

**Comment by jmcclena** [comment]
Thanks @pabloprf. It seems that the `#SBATCH --exclusive` line seems to be preventing the job from running. Is there a way to turn it off?

**Comment by pabloprf** [comment]
Oh interesting! 
Can you remove the line that's adding that `--exclusive` in `misc_tools/FARMINGtools.py` and check if it works? we may need to adjust that.

**Comment by jmcclena** [comment]
Thanks @pabloprf. removing the `--exclusive` logic fixed the run. I'll email omega support to see if they know why that happens.

**Comment by pabloprf** [comment]
Thanks for checking this. I didn't know this could break in certain computers so I will add the `--exclusive` option as part of the config file instead of harcoded. This change will be available in our next release that we are working on.

**Comment by pabloprf** [comment]
This has been solved in `development` commit 27f3bf9ac0546d1a1548c29c9c84b05fe0691b25
The use of node exclusivity is now part of the slurm settings of the config file, e.g.:
```
    "engaging": {
        "machine":          "eofe8.mit.edu", 
        "username":         "pablorf",
        "scratch":          "/pool001/pablorf/scratch/",
        "slurm": {
            "partition":    "sched_mit_psfc",
            "exclusive":    true
        }
    },
```

**Comment by jmcclena** [comment]
Thanks @pabloprf!
